examples/laser/inputs_chirp

```diff
@@ -0,0 +1,32 @@
+max_step = 1
+hipace.dt = 70.e-6/clight
+hipace.verbose=3
```

Comment:
The rest of this input file consistently has a space before and after each `=` sign. Could also be done for prob_lo and prob_hi.
```suggestion
hipace.verbose = 3
```